test/functional/ios/search_context/find_by_ios_predicate_tests.py

```diff
@@ -35,6 +35,10 @@ def test_find_element_by_value(self):
         self.driver.find_element_by_ios_predicate('wdValue == "Buttons"')
 
     def test_find_element_by_isvisible(self):
+        if get_xcode_ver() == '11.3':
+            # TODO Remove skip when it's fixed
```

Comment:
hm, interesting.
I checked the predicate on my local, then it worked expectedly. Xcode 11.3 and iOS 13.3. It might depend on Xcode/iOS side.

```
> @driver.find_elements :predicate, 'wdValue == "Buttons" AND visible == 0'
=> []
> @driver.find_element :predicate, 'wdValue == "Buttons" AND visible == 1'
=> #<Selenium::WebDriver::Element:0x..fa67af99212304fc id="32000000-0000-0000-A80D-010000000000">
```

Reply:
Thanks for your info.
I'll investigate more.

Reply:
[Notes]
Below environment works.
So as a result, its difference is client?...
Will look into communications between client and server.

### Environments
* iPhone 8, iOS 13.3 (Same to python's one)
* [Appium ver] 1.16.0-beta.3

### Commands and test codes

```
$bundle exec rake test:func:ios TESTS=test/functional/ios/ios/search_context_test.rb
```

```
def test_predicate
  e = @@driver.find_element :predicate, 'wdName == "Buttons" AND visible == 1'
  assert_equal 'Buttons', e.name
end
```

Reply:
Resolved by https://github.com/appium/appium/issues/13745#issuecomment-568625153